File: backend/src/main/java/org/jdk/project/dto/config/ComplaintDto.java
package org.jdk.project.dto.config;

import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class ComplaintDto {
  private Long id;
  private String userName;
  private String roomCode;
  private String deptName;
  private String cardService;
  private String cardNo;
  private String complaintContent;
  private String complaintFeedback;
  private String createTime;
}

File: backend/src/main/kotlin/org/jdk/project/dto/config/ComplaintDto.kt
package org.jdk.project.dto.config

data class ComplaintDto(
    var id: Long? = null,
    var userName: String? = null,
    var roomCode: String? = null,
    var deptName: String? = null,
    var cardService: String? = null,
    var cardNo: String? = null,
    var complaintContent: String? = null,
    var complaintFeedback: String? = null,
    var createTime: String? = null,
)
